integration-tests/include-exclude/src/test/java/io/quarkiverse/openapi/generator/it/IncludeExcludeTest.java

```diff
@@ -1,24 +1,24 @@
 package io.quarkiverse.openapi.generator.it;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
 import org.junit.jupiter.api.Test;
 
 import io.quarkus.test.junit.QuarkusTest;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatCode;
-
 @QuarkusTest
-class IgnoreTest {
+class IncludeExcludeTest {
```

Comment:
This test is hard to understand. I think it's better to separate it into 2 different modules (include and exclude), so you can test each property separately. Sorry, I should have realized that sooner :(

Reply:
haha sure, no worries!